lib/src/main/kotlin/app/cash/kfsm/State.kt

```diff
@@ -0,0 +1,25 @@
+package app.cash.kfsm
+
+open class State(transitionsFn: () -> Set<State>) {
+
+  /** all states that can be transitioned to directly from this state */
+  val transitions: Set<State> by lazy { transitionsFn() }
```

Comment:
I wonder if there is another name for this. These aren't `Transition`s, but states reachable by a single transition.
`directlyConnectedStates` is accurate but a bit long.

Reply:
Nice!